nixos/modules/system/boot/kernel.nix

```diff
@@ -105,7 +105,8 @@ in
             foo = true;                 # (may be checked by other NixOS modules, optional)
           };
 
-          extraConfig = "CONFIG_FOO y"; # extra configuration options in string form
+          extraConfig = "FOO y";        # extra configuration options in string form
+                                        # (without the CONFIG_ prefix, optional)
```

Comment:
is it optional or does it break the config ?

Reply:
Fixed.